README.md

```diff
@@ -60,6 +60,114 @@ $ coverage-badge -f -o coverage.svg
 
 ## Resources
 
+<details>
+<summary><strong>Terraform Migration</strong></summary>
+
+
+Documents how environment variables and secrets are currently handled via Ansible and S3 (legacy), and how they will be managed after migrating to Terraform/ECS. It also includes practical lookup references to support migration.
+
+#### 1. Current Setup: Ansible + S3
+
+**How It Works**
+- **Storage:** All secrets & variables reside in S3 (`tlt-secure-settings`) organized by environment (`dev`, `prod`, etc.).
+- **Loading:** The Ansible playbook (`django_deploy.yml` in `tlt-ops/ansible/`) loads these with the `huit-at.s3-vars` role, based on the target environment.
+- **Templating:** Templates like `secure.py.j2` or `.env.j2` are rendered with secret values and placed into the Django project.
+- **Environment selection:** The deployment command you run specifies which S3 object gets loaded.
+- **Injection:** Both secrets (e.g., `DJANGO_SECRET_KEY`, `db_default_password`) and config (e.g., `redis_host`) are injected this way.
+
+#### 2. Migration Plan: Terraform + ECS
```

Comment:
Echoing what Artie stated previously, I think we should move the Migration plan to a separate document and things related to secrets/param references to k459. From my understanding we usually keep extra/misc documentation related but may not be needed in the repo in other locations(Google docs, ADR, Spikes, k459, Runbook repo/doc, etc)